docs/config_entries_config_flow_handler.md

```diff
@@ -11,6 +11,10 @@ When instantiating the handler, Home Assistant will make sure to load all depend
 
 ## Updating the manifest
 
+:::info
+All code in `config_flow.py` must covered by tests to be accepted in a PR. The [initial scaffolding](creating_component_index) includes examples to start writing your [tests](development_testing.md). 
+:::
+
```

Comment:
I don't think that we have to have this info box if we also have the paragraph below.
```suggestion
```